src/Discord.Net.Core/Entities/Permissions/ChannelPermissions.cs

```diff
@@ -82,6 +82,8 @@ public static ChannelPermissions All(IChannel channel)
         public bool UseVAD => Permissions.GetValue(RawValue, ChannelPermission.UseVAD);
         /// <summary> If <c>true</c>, a user may use priority speaker in a voice channel. </summary>
         public bool PrioritySpeaker => Permissions.GetValue(RawValue, ChannelPermission.PrioritySpeaker);
+        /// <summary> If <c>true</c>, a user may stream video in a voice channel. </summary>
+        public bool Stream => Permissions.GetValue(RawValue, ChannelPermission.Stream);
```

Comment:
Could you add `Stream` to the relevant static read-only fields at the top of this file?

Reply:
Hmm. Could that not be constructed from the enums instead of hardcoded?

Reply:
It could be. I believe this was done just for conciseness.

Reply:
Well, unless I'm told otherwise, I'll keep it the way it is for consistency with other places... :)

Reply:
Could you please also update this line in GuildPermissions?

https://github.com/discord-net/Discord.Net/blob/ff0fea98a65d907fbce07856f1a9ef4aebb9108b/src/Discord.Net.Core/Entities/Permissions/GuildPermissions.cs#L15